pkg/blackduck/ctl_blackduck.go

```diff
@@ -205,17 +204,21 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().StringVar(&ctl.AuthCustomCAFilePath, "auth-custom-ca-file-path", ctl.AuthCustomCAFilePath, "Absolute path to a file for the Custom Auth CA for Black Duck")
 	cmd.Flags().StringVar(&ctl.Type, "type", ctl.Type, "Type of Black Duck")
```

Comment:
I don't think `type` is needed in `native` command as well?

Reply:
It could be needed if you are created blackducks to be used with opssight.

Reply:
Sure, but `type` is something that is set for the custom resource's spec, rather than any resources which is what `native` spits out, isn't that right?

Reply:
Yep, it appears to be used only on the CRD.  I'll remove this one as well